psycogreen/gevent.py

```diff
@@ -15,6 +15,12 @@
 from gevent.socket import wait_read, wait_write
 
 
+def is_psycopg_patched():
+    """Return True if psycopg is patched, False otherwise"""
+    if not hasattr(extensions, 'set_wait_callback'):
```

Comment:
Why this?

Reply:
That's a typo, it should be `get_wait_callback` 😅

Reply:
Yes, but why it shouldn't be there?

Reply:
To prevent an exception, so it's safe to call this method in any situation (even if postgresql does not support extensions and it already raised an error when trying to patch)

Reply:
"extensions" is a part of psycopg2, not of postgres, and is always there. get_wait_callback has been there since 2010.

As far as I can see this method can be safely implemented as `return bool(psycopg2.extensions.get_wailt_callback())`.

Being it so trivial, I don't think I want to extend the API for it and receive the burden of its maintenance, so I prefer not merging this patch. Anyway thank you very much for the suggestion

Reply:
I understand, thank you for your work